app/pages/reservationEdit/ReservationEditPageSelector.js

```diff
@@ -29,7 +29,13 @@ const resourceSelector = createSelector(
   (resources, resourceId) => resources[resourceId]
 );
 
+const isFetchingSelector = createSelector(
+  resourceSelector,
+  resource => !resource
+);
+
```

Comment:
This won't work if the reservation is not found. Which will avoid the "Varausta ei löytynyt." message to appear.

Maybe you could use "reservationsGetIsActiveSelector || resourceGetIsActiveSelector"